### PR TITLE
Create registry_set_scr_file_executed_by_rundll32.yml

### PR DESCRIPTION
**Pull request by jstnk9**
First PR. Let me know if I've to change anything please :)

### CODE REVIEW

**Comment by frack113** [comment]
Hi, 
when use graphical interface image is `rundll32.exe`
image

So will trigger when user change the screensaver

**Comment by jstnk9** [comment]
@frack113 thanks for the observation!

What about if we add a filter to excude Details from System32 and SysWOW64? Checking telemetry from a large number of endpoints, I can see that a very high percentage, if not almost all, of the value comes from those paths (system32 and syswow64).

Next image is just an example on my lab

**Excluding system32:**
image

query:

```
(winlog.event_data.EventType:"SetValue" AND winlog.event_data.Image:"*\\rundll32.exe") AND (winlog.event_data.TargetObject:"*\\Control Panel\\Desktop\\SCRNSAVE.EXE*" AND winlog.event_data.Details:*.scr) AND NOT (winlog.event_data.Details:"C:\\Windows\\System32\\*")
```

**Without excluding**
image

query:

```
(winlog.event_data.EventType:"SetValue" AND winlog.event_data.Image:"*\\rundll32.exe") AND (winlog.event_data.TargetObject:"*\\Control Panel\\Desktop\\SCRNSAVE.EXE*" AND winlog.event_data.Details:*.scr)
```

**Comment by jstnk9** [comment]
new changes:

- deleted ScreenSaveActive registry key
- deleted ScreenSaveTimeOut key
- added filter to check if system32 or syswow64 on the registry key

File: rules/windows/registry/registry_set/registry_set_scr_file_executed_by_rundll32.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        Image|endswith: '\rundll32.exe'
+    registry:
+        TargetObject|contains: '\Control Panel\Desktop\SCRNSAVE.EXE'
+        Details|endswith: '.scr'
+    filter:
+        Details|contains: 
+        - 'C:\Windows\System32\'
+        - 'C:\Windows\SysWOW64\'
+        - 'C:\Windows\system32\'
+        - 'C:\Windows\SysWow64\'
```
Comment (frack113) [inline rules/windows/registry/registry_set/registry_set_scr_file_executed_by_rundll32.yml:27]: Hi,
shoud be only `C:\Windows\System32` and `C:\Windows\SysWOW64`.
Elk use for text `keyword` type casa-sensitive or `text` type case-insensitive.
You have to use some sigmac option to deal with.
I kown elastic is a pain in the neck.

**Comment by jstnk9** [comment]
New commit with 2 paths only :)

**Review by frack113 — APPROVED** [review]
No FP with aurora tests

**Review by Neo23x0 — APPROVED** [review]

**Pull request merged**